build-system/server/new-server/transforms/cdn/cdn-transform.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
```

Comment:
2020 because it's a rename of a previous file that was handled by git as delete/create. https://github.com/ampproject/amphtml/blob/main/build-system/server/new-server/transforms/scripts/scripts-transform.ts